clients/cobol-lsp-vscode-extension/src/services/copybook/CopybooksCompletionProvider.ts

```diff
@@ -51,7 +52,10 @@ const isSQLCopyStatement: CopyStatementParser = (statement: string) => {
 };
 
 export class CopybooksCompletionProvider implements CompletionItemProvider {
-  constructor(private cds?: CopybookDownloadService) {}
+  constructor(
+    private outputChannel?: OutputChannel,
+    private cds?: CopybookDownloadService,
```

Comment:
```suggestion
    private cds?: CopybookDownloadService,
    private outputChannel?: OutputChannel,
```
and remove the unnecessary test changes please...